index.bs

```diff
@@ -1472,12 +1473,17 @@ To <dfn>validate a suffix</dfn> |suffix|, run the following steps:
 
 <div class="note domintro">
   : |handle| = await window . {{Window/showDirectoryPicker()}}
+  : |handle| = await window . {{Window/showDirectoryPicker()}}({ {{OpenDirectoryPickerOptions/mode}}: 'read' })
   :: Shows a directory picker that lets the user select a single directory, returning a handle for
     the selected directory.
+
+  : handles = await window . {{Window/showDirectoryPicker()}}({ {{OpenDirectoryPickerOptions/mode}}: 'readwrite' })
+  :: Shows a directory picker that lets the user select a single directory, returning a handle for
+    the selected directory. The website can ask for read and write permissions on this handle in one step.
```

Comment:
"the website can ask" seems weird, it is more that the user agent can combine prompts or something like that...